src/astro/databases/postgres.py

```diff
@@ -24,4 +27,50 @@ def hook(self):
     @property
     def default_metadata(self) -> Metadata:
         schema = self.hook.schema
-        return Metadata(schema=schema)
+        return Metadata(database=schema)
+
+    def schema_exists(self, schema):
+        created_schemas = self.hook.run(
+            "SELECT schema_name FROM information_schema.schemata;",
+            handler=lambda x: [y[0] for y in x.fetchall()],
+        )
+        return schema.upper() in [c.upper() for c in created_schemas]
```

Comment:
Would it be worth to use SQL-only to check if this table exists? E.g.:
`SELECT schema_name FROM information_schema.schemata WHERE lower(schema_name) = lower({})`

Reply:
Ooooo that would be great so we're not loading all of this stuff locally!